packages/forms/frontend/sirius-components-widget-reference/src/components/ModelBrowserTreeView.tsx

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { TreeView } from '@eclipse-sirius/sirius-components-trees';
+import { makeStyles } from '@material-ui/core/styles';
+import { useState } from 'react';
+import { ModelBrowserFilterBar } from './ModelBrowserFilterBar';
+import { ModelBrowserTreeViewProps } from './ModelBrowserTreeView.types';
+
+const useTreeStyle = makeStyles((_) => ({
+  ul: {
+    width: 'inherit',
+    minWidth: 'fit-content',
```

Comment:
Those properties are an odd way to layout things, those are still CSS draft specifications

Reply:
I deleted them, there were not used.